lib/ohai/plugins/windows/virtualization.rb

```diff
@@ -44,6 +44,14 @@
       virtualization[:system] = "kvm"
       virtualization[:role] = "guest"
       virtualization[:systems][:kvm] = "guest"
+    when "American Megatrends Inc."
```

Comment:
I think what you want here is a match with a regex. There's two ways to do that with ruby, but the easiest here is:

if bios[0]["version"] =~ /VRTUAL -/

Here's a nice doc with all your options:
https://ruby-doc.org/core-2.2.0/Regexp.html